src/Symfony/Component/Console/Helper/OutputWrapperInterface.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Console\Helper;
+
+interface OutputWrapperInterface
+{
+    public const TAG_OPEN_REGEX = '[a-z](?:[^\\\\<>]*+ | \\\\.)*';
+    public const TAG_CLOSE_REGEX = '[a-z][^<>]*+';
+
+    public function wrap(string $text, int $width, string $break = "\n"): string;
```

Comment:
```suggestion
    /**
     * @param positive-int|0 $width
     */
    public function wrap(string $text, int $width, string $break = "\n"): string;
```

It may not be worth adding an assertion for it, but I don't think the current code really expects a negative value either